src/components/LoginForm.vue

```diff
@@ -21,36 +23,43 @@
     <div class="grid--email my-4">
       <base-input
         v-model="email"
+        data-testid="testEmailTextInput"
         required
         autocomplete="email"
         :placeholder="lang.email"
         size="xlarge"
         type="email"
-        data-cy="loginForm.email"
+        data-testid="loginForm.email"
```

Comment:
We need to remove these duplicates, because it breaks the build

Reply:
Will do. They were pre-existing and I didn't catch them all.

Reply:
OK. I think I caught them all with the latest commit.